src/main/java/org/opensearch/neuralsearch/query/HybridQueryScorer.java

```diff
@@ -187,7 +189,7 @@ public int docID() {
      */
     public float[] hybridScores() throws IOException {
         float[] scores = new float[numSubqueries];
-        DisiWrapper topList = subScorersPQ.topList();
+        DisiWrapper topList = getSubMatches();
```

Comment:
this is main change for this PR

Reply:
Could you explain here what is the difference between `subScorersPQ.topList();` and `getSubMatches()`.

Reply:
makes sense, let me put that as a comment